<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-HOME" failures="1" tests="10" name="Test" time="12.566" errors="0" timestamp="2023-03-06T02:07:09 IST">
  <testcase classname="com.CMS.TestCases.LoginPageTestCases" name="successfulLoginTest" time="1.592"/>
  <testcase classname="com.CMS.TestCases.LoginPageTestCases" name="successfulLoginTest" time="0.482"/>
  <testcase classname="com.CMS.TestCases.LoginPageTestCases" name="successfulLoginTest" time="0.505"/>
  <testcase classname="com.CMS.TestCases.LoginPageTestCases" name="successfulLoginTest" time="0.459"/>
  <testcase classname="com.CMS.TestCases.LoginPageTestCases" name="successfulLoginTest" time="0.285"/>
  <testcase classname="com.CMS.TestCases.LoginPageTestCases" name="successfulLoginTest" time="0.553"/>
  <testcase classname="com.CMS.TestCases.LoginPageTestCases" name="successfulLoginTest" time="1.065"/>
  <testcase classname="com.CMS.TestCases.LoginPageTestCases" name="successfulLoginTest" time="1.153"/>
  <testcase classname="com.CMS.TestCases.LoginPageTestCases" name="successfulLoginTest" time="1.336"/>
  <testcase classname="com.CMS.TestCases.LoginPageTestCases" name="forgetPasswordTest" time="1.01">
    <failure type="java.lang.AssertionError" message="expected [Enter your Login ID to receive your password in mail.User Name] but found [Enter your Login ID to receive your password in mail.
User Name]">
      <![CDATA[java.lang.AssertionError: expected [Enter your Login ID to receive your password in mail.User Name] but found [Enter your Login ID to receive your password in mail.
User Name]
at com.CMS.TestCases.LoginPageTestCases.forgetPasswordTest(LoginPageTestCases.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- forgetPasswordTest -->
</testsuite> <!-- Test -->
